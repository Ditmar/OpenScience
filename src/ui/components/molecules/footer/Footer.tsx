import { useTheme } from '@mui/material/styles';
import classNames from 'classnames';
import styles from './Footer.module.scss';
import LogoFooter from '../../atoms/logo-footer/LogoFooter';
import Variables from '../../atoms/label-variables/Variables';
import LabelResourcesPage from '../../atoms/label-resources-page/LabelResourcesPage';
import DividerVariables from '../../atoms/divider-variables/DividerVariables';
import logo from '../../../../assets/icons/logo.svg?raw';
import type { IFooterProps } from './types/IProps';
import Location from '../location-variable/Location';

function Footer({ locationText, followUsText, size = 'medium' }: IFooterProps) {
  const theme = useTheme();

  return (
    <div
      className={classNames(styles.footer, styles[`footer--${size}`])}
      style={{
        color: theme.palette.text.primary,
        backgroundColor: theme.palette.background.default,
        borderColor: theme.palette.background.default,
      }}
    >
      <div className={styles['footer__logo-section']}>
        <LogoFooter icon={logo} color="secondary" />
      </div>

      <div className={classNames(styles.footer__variables, styles['footer__vertical-line'])}>
        <div className={styles.footer__divider}>
          <DividerVariables />
        </div>

        <div className={classNames(styles.footer__section, styles['footer__section--left'])}>
          <div className={styles.footer__group}>
            <div className={styles.footer__label}>
              <LabelResourcesPage text={followUsText} />
            </div>
            <div className={styles.footer__section1}>
              <div className={classNames(styles.footer__text, styles['footer__text--shift-right'])}>
                <Variables text="ARTÍCULOS" colorVariant="primary" />
              </div>
              <div className={styles.footer__text}>
                <Variables text="VOLUMENES" colorVariant="primary" />
              </div>
            </div>
          </div>
        </div>

        <div className={classNames(styles.footer__section, styles['footer__section--right'])}>
          <div className={styles.footer__group}>
            <div className={styles.footer__label}>
              <LabelResourcesPage text="PÁGINA" />
            </div>
            <div className={styles.footer__text}>
              <Variables text={locationText} colorVariant="primary" />
            </div>
          </div>
        </div>
      </div>

      <div className={styles.footer__location} />
      <Location locationText={locationText} followUsText={followUsText} size={size} />
    </div>
  );
}

export default Footer;
