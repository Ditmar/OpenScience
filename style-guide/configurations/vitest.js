const { TEST_FILES } = require('./constants.js');
module.exports = {
  overrides: [
    {
      files: TEST_FILES,
      plugins: ['vitest'],
      extends: ['plugin:vitest/recommended'],
      rules: {
        'import/no-extraneous-dependencies': [
          'off',
          { devDependencies: ['**/*.test.[jt]s?(x)', '**/*.spec.[jt]s?(x)'] },
        ],
      },
    },
  ],
};
