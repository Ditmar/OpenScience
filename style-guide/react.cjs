const { resolve } = require('node:path');
const project = resolve(process.cwd(), 'tsconfig.json');

module.exports = {
  env: {
    browser: true,
    es2021: true,
    es6: true,
    jest: true,
  },
  extends: [
    require.resolve('./rules/base'),
    require.resolve('./configurations/_base'),
    'plugin:react-hooks/recommended',
    'airbnb',
    'plugin:storybook/recommended',
    require.resolve('./configurations/typescript'),
    require.resolve('./configurations/vitest'),
  ],
  parserOptions: {
    project,
    babelOptions: {
      presets: ['@babel/preset-react'],
    },
  },
  plugins: ['react-refresh'],
  settings: {
    react: {
      version: 'detect',
    },
  },
  globals: {
    JSX: true,
  },
  rules: {
    'react/react-in-jsx-scope': 'off',
    'react/button-has-type': 'off',
    'react-refresh/only-export-components': [
      'warn',
      { allowExportNames: ['meta', 'links', 'headers', 'loader', 'action'] },
    ],
  },
};
