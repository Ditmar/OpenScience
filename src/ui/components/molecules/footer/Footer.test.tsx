import { render, screen } from '@testing-library/react';
import { describe, test, expect } from 'vitest';
import Footer from './Footer';

describe('Footer component', () => {
  test('renders static content', () => {
    render(<Footer locationText="UNIPOL.EDU.BO" followUsText="RECURSOS" />);
    expect(screen.getByText('ARTÍCULOS')).toBeDefined();
    expect(screen.getByText('VOLUMENES')).toBeDefined();
    expect(screen.getByText('PÁGINA')).toBeDefined();
  });

  test('renders at least two SVG icon containers (Location)', () => {
    const { container } = render(<Footer locationText="UNIPOL.EDU.BO" followUsText="RECURSOS" />);
    const icons = container.querySelectorAll('span[aria-hidden="true"]');
    expect(icons.length).toBeGreaterThanOrEqual(2);
  });

  test('applies correct size class: small', () => {
    const { container } = render(
      <Footer locationText="UNIPOL.EDU.BO" followUsText="RECURSOS" size="small" />,
    );
    const root = container.firstChild as HTMLElement;
    expect(root.className.includes('footer--small')).toBeTruthy();
  });

  test('applies correct size class: large', () => {
    const { container } = render(
      <Footer locationText="UNIPOL.EDU.BO" followUsText="RECURSOS" size="large" />,
    );
    const root = container.firstChild as HTMLElement;
    expect(root.className.includes('footer--large')).toBeTruthy();
  });

  test('matches DOM structure snapshot', () => {
    const { asFragment } = render(
      <Footer locationText="UBICANOS" followUsText="SIGUENOS EN" size="medium" />,
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
