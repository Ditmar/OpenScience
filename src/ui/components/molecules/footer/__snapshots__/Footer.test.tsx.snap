// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Footer component > matches DOM structure snapshot 1`] = `
<DocumentFragment>
  <div
    class="_footer_70971b _footer--medium_70971b"
    style="color: rgba(0, 0, 0, 0.87); background-color: rgb(255, 255, 255); border-color: rgb(255, 255, 255);"
  >
    <div
      class="_footer__logo-section_70971b"
    >
      <div
        class="_logo_336c6d _logo--secondary_336c6d"
      >
        <span
          class="_logo-icon_336c6d"
          data-testid="logo_icon"
        >
          <!--?xml version="1.0" standalone="no"?-->
          <svg
            preserveAspectRatio="xMidYMid meet"
            version="1.0"
            viewBox="0 0 578.000000 288.000000"
            xmlns="http://www.w3.org/2000/svg"
          >
            <g
              fill="#000000"
              stroke="none"
              transform="translate(0.000000,288.000000) scale(0.100000,-0.100000)"
            >
              <path
                d="M2351 2476 c-8 -9 -11 -85 -9 -257 l3 -244 180 -5 180 -5 0 -25 c0
-24 -3 -25 -80 -32 -44 -4 -91 -10 -105 -13 -24 -6 -24 -7 -5 -18 23 -14 452
-17 500 -4 47 13 11 26 -94 34 l-91 6 0 28 c0 24 4 29 25 29 23 0 25 4 25 45
l0 45 -240 0 -240 0 0 195 0 195 370 0 370 0 0 -50 c0 -47 2 -50 25 -50 26 0
26 1 23 68 l-3 67 -411 3 c-341 2 -413 0 -423 -12z m419 -452 c0 -9 -5 -12
-12 -8 -8 5 -9 1 -4 -12 6 -15 5 -16 -5 -7 -10 10 -9 15 1 28 16 19 20 19 20
-1z m20 -14 c0 -11 -4 -20 -9 -20 -5 0 -7 9 -4 20 3 11 7 20 9 20 2 0 4 -9 4
-20z"
              />
              <path
                d="M2915 2327 c-3 -7 -4 -91 -3 -187 l3 -175 73 -3 c59 -2 72 0 72 13 0
11 -11 15 -40 15 l-40 0 0 155 0 155 215 0 215 0 0 -155 0 -155 -35 0 c-25 0
-35 -4 -35 -15 0 -13 12 -15 68 -13 l67 3 0 185 0 185 -278 3 c-220 2 -279 0
-282 -11z m553 -180 c2 -12 -3 -17 -17 -17 -12 0 -21 6 -21 13 0 31 32 34 38
4z"
              />
              <path
                d="M3087 2234 c-4 -4 -7 -90 -7 -191 l0 -183 115 0 115 0 -2 188 -3 187
-106 3 c-58 1 -108 -1 -112 -4z m126 -31 c-7 -2 -21 -2 -30 0 -10 3 -4 5 12 5
17 0 24 -2 18 -5z m77 -148 l0 -125 -95 0 -95 0 0 125 0 125 95 0 95 0 0 -125z
m-80 -171 c0 -16 -16 -19 -25 -4 -8 13 4 32 16 25 5 -4 9 -13 9 -21z"
              />
              <path
                d="M4662 1417 c-42 -34 -48 -43 -33 -45 17 -3 131 71 131 84 0 2 -10 4
-22 4 -13 0 -47 -19 -76 -43z"
              />
              <path
                d="M480 1035 c0 -278 0 -285 20 -285 20 0 20 7 20 285 0 278 0 285 -20
285 -20 0 -20 -7 -20 -285z"
              />
              <path
                d="M720 1035 c0 -278 0 -285 20 -285 19 0 20 7 20 246 0 204 2 245 13
235 8 -6 97 -116 198 -246 115 -147 191 -235 202 -235 16 0 17 20 17 285 0
277 -1 285 -20 285 -19 0 -20 -8 -22 -247 l-3 -247 -193 247 c-134 172 -199
247 -213 247 -18 0 -19 -9 -19 -285z"
              />
              <path
                d="M1514 1299 c-65 -25 -120 -78 -152 -147 -21 -47 -24 -65 -20 -131 7
-122 60 -200 168 -250 89 -41 209 -32 298 21 l32 20 0 109 c0 102 -1 109 -20
109 -18 0 -20 -8 -22 -102 l-3 -103 -40 -20 c-50 -25 -171 -25 -227 -1 -83 36
-148 134 -148 225 0 99 62 201 142 234 84 35 195 28 262 -18 28 -19 37 -21 45
-11 14 17 -9 38 -69 65 -60 28 -175 28 -246 0z"
              />
              <path
                d="M2030 1300 c0 -20 5 -20 182 -18 165 3 183 5 186 21 3 16 -11 17
-182 17 -180 0 -186 -1 -186 -20z"
              />
              <path
                d="M2580 1035 c0 -277 1 -285 20 -285 19 0 20 8 22 245 l3 245 191 -242
c104 -134 198 -244 207 -246 16 -3 17 19 17 282 0 278 -1 286 -20 286 -19 0
-20 -8 -22 -244 l-3 -245 -190 244 c-117 151 -197 245 -207 245 -17 0 -18 -20
-18 -285z"
              />
              <path
                d="M3250 1035 c0 -278 0 -285 20 -285 20 0 20 7 20 285 0 278 0 285 -20
285 -20 0 -20 -7 -20 -285z"
              />
              <path
                d="M3490 1300 c0 -19 7 -20 185 -20 178 0 185 1 185 20 0 19 -7 20 -185
20 -178 0 -185 -1 -185 -20z"
              />
              <path
                d="M4040 1301 c0 -20 5 -21 135 -21 148 0 182 -8 225 -54 34 -37 40
-137 11 -183 -36 -57 -69 -67 -228 -71 l-143 -4 0 -109 c0 -102 1 -109 20
-109 19 0 20 7 20 95 l0 96 112 -3 112 -3 65 -92 c51 -71 71 -93 89 -93 22 0
21 4 -24 68 -26 37 -58 81 -71 99 -24 31 -24 31 -3 38 30 10 79 53 96 87 21
41 20 130 -3 176 -11 23 -36 48 -67 67 -48 29 -53 30 -198 33 -145 4 -148 3
-148 -17z"
              />
              <path
                d="M4630 1035 c0 -278 0 -285 20 -285 20 0 20 7 20 285 0 278 0 285 -20
285 -20 0 -20 -7 -20 -285z"
              />
              <path
                d="M5018 1273 c-12 -27 -68 -149 -125 -273 -58 -124 -107 -231 -110
-238 -4 -10 1 -13 18 -10 20 3 39 38 139 255 63 139 117 254 120 257 3 2 57
-111 121 -252 98 -216 120 -258 139 -260 28 -5 37 -28 -113 300 -108 237 -125
268 -145 268 -18 0 -28 -10 -44 -47z"
              />
              <path
                d="M2030 1040 c0 -19 7 -20 185 -20 178 0 185 1 185 20 0 19 -7 20 -185
20 -178 0 -185 -1 -185 -20z"
              />
              <path
                d="M3490 1040 c0 -19 7 -20 185 -20 178 0 185 1 185 20 0 19 -7 20 -185
20 -178 0 -185 -1 -185 -20z"
              />
              <path
                d="M2030 765 c0 -13 27 -15 185 -15 158 0 185 2 185 15 0 13 -27 15
-185 15 -158 0 -185 -2 -185 -15z"
              />
              <path
                d="M3490 765 c0 -13 27 -15 185 -15 158 0 185 2 185 15 0 13 -27 15
-185 15 -158 0 -185 -2 -185 -15z"
              />
              <path
                d="M1710 650 c0 -6 32 -10 76 -10 74 0 77 -1 111 -35 33 -33 35 -39 30
-81 -8 -72 -39 -97 -127 -102 l-70 -4 0 56 c0 32 -4 56 -10 56 -7 0 -10 -24
-8 -62 3 -72 8 -75 106 -63 77 10 125 53 126 117 2 91 -55 138 -168 138 -37 0
-66 -4 -66 -10z"
              />
              <path
                d="M2020 650 c0 -6 33 -10 80 -10 47 0 80 4 80 10 0 6 -33 10 -80 10
-47 0 -80 -4 -80 -10z"
              />
              <path
                d="M2372 650 c-12 -5 -27 -21 -33 -34 -18 -38 9 -72 72 -90 63 -19 79
-32 79 -62 0 -42 -76 -59 -132 -30 -16 8 -28 10 -28 5 0 -18 72 -41 108 -35
102 16 93 108 -13 135 -70 17 -93 52 -58 86 14 14 29 16 77 12 56 -5 59 -4 36
8 -28 16 -74 18 -108 5z"
              />
              <path
                d="M2570 530 c0 -80 4 -130 10 -130 6 0 10 50 10 130 0 80 -4 130 -10
130 -6 0 -10 -50 -10 -130z"
              />
              <path
                d="M2702 650 c-12 -5 -27 -21 -32 -36 -16 -40 9 -70 71 -88 63 -19 79
-32 79 -62 0 -42 -76 -59 -132 -30 -16 8 -28 10 -28 5 0 -18 72 -41 108 -35
102 16 93 108 -13 135 -71 18 -93 52 -55 86 16 14 30 17 77 12 54 -5 56 -5 33
8 -28 16 -74 18 -108 5z"
              />
              <path
                d="M2860 650 c0 -5 20 -10 45 -10 l45 0 0 -121 c0 -77 4 -118 10 -114 6
4 10 55 10 121 l0 114 45 0 c25 0 45 5 45 10 0 6 -40 10 -100 10 -60 0 -100
-4 -100 -10z"
              />
              <path
                d="M3110 650 c0 -6 33 -10 80 -10 47 0 80 4 80 10 0 6 -33 10 -80 10
-47 0 -80 -4 -80 -10z"
              />
              <path
                d="M3350 530 c0 -159 18 -177 22 -23 l3 108 50 -88 c28 -48 52 -87 55
-87 3 0 28 39 55 86 l50 86 3 -98 c4 -153 22 -141 22 14 0 73 -3 132 -7 132
-5 0 -32 -43 -62 -95 -29 -52 -57 -95 -61 -95 -4 0 -32 43 -62 95 -29 52 -56
95 -60 95 -5 0 -8 -58 -8 -130z"
              />
              <path
                d="M3720 539 c-67 -148 -64 -139 -51 -139 6 0 11 4 11 9 0 5 23 58 51
117 l51 109 59 -120 c33 -66 57 -109 54 -95 -6 35 -103 240 -112 240 -5 0 -33
-55 -63 -121z"
              />
              <path
                d="M3990 653 c-52 -19 -68 -79 -27 -105 12 -8 44 -20 70 -28 59 -16 75
-41 52 -77 -15 -22 -22 -24 -63 -20 -26 2 -58 7 -72 11 -14 4 -7 -3 16 -15 70
-37 144 -10 144 51 0 33 -25 54 -75 66 -47 10 -75 32 -75 58 0 37 23 49 86 44
50 -4 55 -3 39 9 -19 13 -67 16 -95 6z"
              />
              <path
                d="M2053 533 c26 -2 67 -2 90 0 23 2 2 3 -48 3 -49 0 -68 -1 -42 -3z"
              />
              <path
                d="M3148 533 c23 -2 61 -2 85 0 23 2 4 4 -43 4 -47 0 -66 -2 -42 -4z"
              />
              <path
                d="M2015 410 c3 -5 39 -10 80 -10 41 0 77 5 80 10 4 6 -25 10 -80 10
-55 0 -84 -4 -80 -10z"
              />
              <path
                d="M3110 410 c0 -6 33 -10 80 -10 47 0 80 4 80 10 0 6 -33 10 -80 10
-47 0 -80 -4 -80 -10z"
              />
            </g>
          </svg>
        </span>
      </div>
    </div>
    <div
      class="_footer__variables_70971b _footer__vertical-line_70971b"
    >
      <div
        class="_footer__divider_70971b"
      >
        <div
          class="divider-bar"
        />
      </div>
      <div
        class="_footer__section_70971b _footer__section--left_70971b"
      >
        <div
          class="_footer__group_70971b"
        >
          <div
            class="_footer__label_70971b"
          >
            <div
              class="resources-page"
            >
              <span
                class="text"
              >
                RECURSOS
              </span>
            </div>
          </div>
          <div
            class="_footer__section1_70971b"
          >
            <div
              class="_footer__text_70971b _footer__text--shift-right_70971b"
            >
              <div
                class="variables variables--primary"
              >
                <span
                  class="variables__articulos"
                >
                  ART√çCULOS
                </span>
              </div>
            </div>
            <div
              class="_footer__text_70971b"
            >
              <div
                class="variables variables--primary"
              >
                <span
                  class="variables__articulos"
                >
                  VOLUMENES
                </span>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="_footer__section_70971b _footer__section--right_70971b"
      >
        <div
          class="_footer__group_70971b"
        >
          <div
            class="_footer__label_70971b"
          >
            <div
              class="resources-page"
            >
              <span
                class="text"
              >
                P√ÅGINA
              </span>
            </div>
          </div>
          <div
            class="_footer__text_70971b"
          >
            <div
              class="variables variables--primary"
            >
              <span
                class="variables__articulos"
              >
                UNIPOL.EDU.BO
              </span>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="_footer__location_70971b"
    />
    <div
      class="_location_3ac954 _location--medium_3ac954"
      role="contentinfo"
      style="color: rgba(0, 0, 0, 0.87);"
    >
      <div
        class="_location__section_3ac954"
      >
        <p
          class="MuiTypography-root MuiTypography-body2 _location__text_3ac954 css-1o4rdrb-MuiTypography-root"
        >
          UNIPOL.EDU.BO
        </p>
        <span
          aria-hidden="true"
          class="_location__icon_3ac954"
        >
          <svg
            fill="none"
            height="44"
            viewBox="0 0 51 44"
            width="51"
            xmlns="http://www.w3.org/2000/svg"
          >
            

            <g
              clip-path="url(#clip0_807_249)"
            >
              

              <path
                d="M25.51 34.5007C23.9664 35.7139 22.4495 36.9053 20.9339 38.0987C18.5342 39.9889 16.1366 41.8804 13.7376 43.7706C13.3477 44.0775 12.9891 44.0736 12.5933 43.762C8.53852 40.5698 4.48578 37.3756 0.427059 34.1886C0.107595 33.937 0 33.6314 0 33.2448C0.00464916 30.0947 0.00265667 26.9446 0.00265667 23.7945C0.00265667 17.4172 0.00265667 11.04 0.00265667 4.66337C0.00265667 4.56458 -0.0019925 4.46579 0.003985 4.367C0.02391 4.0153 0.15143 3.7242 0.498125 3.57667C0.824895 3.43771 1.12111 3.50489 1.39541 3.72157C2.40694 4.52177 3.42046 5.32 4.43397 6.11823C7.26664 8.34958 10.0986 10.5809 12.932 12.8116C13.009 12.8722 13.0907 12.9282 13.1777 12.9921C14.2032 12.184 15.2273 11.3818 16.2429 10.5691C16.3073 10.5177 16.3498 10.386 16.3378 10.3004C15.6816 5.61966 19.0576 0.981117 23.987 0.136128C29.0858 -0.737839 33.9602 2.67307 34.795 7.74498C34.9405 8.62816 34.9863 9.50806 34.8109 10.3926C34.7943 10.4775 34.8402 10.6158 34.9066 10.6692C35.8789 11.447 36.8592 12.2143 37.8542 12.996C38.7468 12.2933 39.6281 11.5991 40.5102 10.905C43.5421 8.51686 46.574 6.12942 49.6039 3.74001C49.8968 3.50884 50.2116 3.41993 50.557 3.59182C50.8884 3.75647 50.9993 4.03836 50.9987 4.40454C50.9934 14.0524 50.9927 23.7003 51 33.3482C51 33.7131 50.8718 33.9666 50.5882 34.1899C46.574 37.3446 42.5651 40.5059 38.5542 43.6646C37.985 44.1131 37.7233 44.1124 37.1508 43.6613C33.4255 40.7285 29.7009 37.7938 25.9755 34.8603C25.8288 34.7444 25.6793 34.6318 25.51 34.5014V34.5007ZM37.0359 41.509C37.0419 41.3753 37.0472 41.3114 37.0472 41.2468C37.0472 32.3886 37.0485 23.531 37.0392 14.6728C37.0392 14.5372 36.9469 14.3607 36.84 14.2731C36.0456 13.625 35.2367 12.9947 34.4304 12.3611C34.3753 12.3177 34.3042 12.2946 34.2311 12.2564C33.4627 14.057 32.4505 15.6917 31.2955 17.2322C29.6324 19.4497 27.9209 21.631 26.2219 23.8222C25.8726 24.2733 25.2622 24.2239 24.8923 23.7497C23.8974 22.4746 22.8958 21.2042 21.9009 19.9285C20.2969 17.8723 18.7255 15.7931 17.5187 13.4722C17.2949 13.0415 17.0983 12.5969 16.8798 12.1392C15.9426 12.8768 15.0208 13.5954 14.1115 14.3284C14.0298 14.3942 13.99 14.5549 13.99 14.6715C13.9847 23.5073 13.9847 32.3432 13.986 41.1797C13.986 41.2679 14.0026 41.3555 14.0146 41.4754C14.0969 41.4194 14.1348 41.397 14.1693 41.37C17.7379 38.5591 21.3065 35.7475 24.875 32.9359C25.3333 32.575 25.6793 32.5658 26.1256 32.9168C28.6561 34.9078 31.1853 36.9 33.7144 38.8923C34.8043 39.7505 35.8942 40.6086 37.0359 41.5083V41.509ZM49.3808 5.99638C49.228 6.11032 49.1397 6.17289 49.0553 6.24007C45.6667 8.90675 42.2802 11.5741 38.8883 14.2362C38.7116 14.3751 38.6591 14.5253 38.6591 14.7393C38.6638 20.4251 38.6631 26.1108 38.6631 31.7972V32.1746C38.8338 32.1746 38.954 32.1779 39.0742 32.1746C39.8812 32.1489 40.6875 32.1179 41.4945 32.0962C42.0205 32.0824 42.3758 32.3735 42.4143 32.8358C42.4509 33.2705 42.136 33.6241 41.6399 33.692C41.4433 33.719 41.2434 33.7355 41.0455 33.7361C40.2638 33.7414 39.4814 33.7381 38.6884 33.7381V41.4708C38.7568 41.4286 38.7953 41.4095 38.8278 41.3832C42.2948 38.6559 45.7611 35.9293 49.2247 33.198C49.2944 33.1427 49.3681 33.0446 49.3728 32.9636C49.3901 32.6409 49.3794 32.3162 49.3794 31.9724C49.2811 31.9974 49.2287 32.0086 49.1782 32.0244C48.2955 32.3017 47.4168 32.5915 46.5295 32.8529C46.0719 32.9879 45.6355 32.7284 45.5127 32.2957C45.3911 31.8663 45.6402 31.4508 46.0971 31.3032C47.0721 30.9897 48.0385 30.6473 49.0234 30.3693C49.3515 30.2765 49.3861 30.1283 49.3854 29.849C49.3794 22.0446 49.3808 14.2401 49.3808 6.43567V5.99638ZM12.3389 24.6988C12.1277 24.7758 11.9311 24.899 11.7265 24.9141C10.6585 24.9951 9.58857 25.0564 8.51793 25.1084C8.04637 25.1315 7.7196 24.839 7.68108 24.405C7.6399 23.9466 7.91221 23.5785 8.38377 23.535C9.29634 23.4507 10.2116 23.3881 11.1268 23.3387C11.5439 23.3163 11.9829 23.2498 12.3415 23.6325C12.3495 23.5291 12.3561 23.4863 12.3561 23.4428C12.3561 20.5127 12.3595 17.5825 12.3475 14.6524C12.3475 14.5181 12.2466 14.3422 12.137 14.2553C8.73976 11.5662 5.33591 8.88568 1.93206 6.2045C1.85302 6.14194 1.76735 6.08727 1.66174 6.01285V25.138C2.26946 24.9405 2.85259 24.7468 3.43972 24.5637C3.58451 24.519 3.74258 24.4781 3.89002 24.4906C4.29317 24.5242 4.5874 24.8285 4.62857 25.2151C4.66776 25.5826 4.42866 25.9297 4.0368 26.066C3.32947 26.313 2.62479 26.5646 1.9128 26.7984C1.69827 26.8689 1.63584 26.9742 1.63651 27.1975C1.64713 29.0409 1.64846 30.885 1.63584 32.7284C1.63385 32.9893 1.72418 33.1447 1.92276 33.3008C5.32993 35.9773 8.73179 38.6598 12.1363 41.3404C12.1908 41.3832 12.2572 41.4102 12.3389 41.4563V24.6988ZM25.585 21.982C26.4385 20.9039 27.2707 19.8659 28.0889 18.8174C29.5707 16.9187 31.0199 14.9975 32.1596 12.8683C33.0263 11.2481 33.5537 9.5838 33.1459 7.70349C32.307 3.83287 28.6203 1.12074 24.5695 1.68516C21.8398 2.06518 19.8393 3.53321 18.6265 5.99507C17.9723 7.32215 17.7007 8.73222 17.9883 10.2062C18.1829 11.2033 18.6139 12.1168 19.0941 13.0066C20.0299 14.742 21.1783 16.3384 22.3784 17.8987C23.4238 19.2574 24.4958 20.597 25.5857 21.9814L25.585 21.982Z"
                fill="#02322C"
              />
              

              <path
                d="M16.4495 23.7297C17.3123 23.9556 18.1757 24.1775 19.0365 24.4093C19.2915 24.4778 19.5439 24.5608 19.7876 24.6603C20.1987 24.8289 20.3887 25.2596 20.2472 25.66C20.0951 26.0894 19.6907 26.2949 19.243 26.1751C18.2328 25.9037 17.2226 25.635 16.2124 25.365C15.8059 25.2563 15.5483 24.8776 15.602 24.4673C15.6558 24.0583 16.0072 23.7685 16.4495 23.729V23.7297Z"
                fill="#02322C"
              />
              

              <path
                d="M33.5525 33.1431C33.1228 33.0219 32.5582 32.8763 32.003 32.7025C31.4783 32.5378 30.9589 32.3534 30.4442 32.1598C29.9421 31.9707 29.7295 31.5394 29.8942 31.0909C30.0523 30.6608 30.5325 30.4889 31.0253 30.6509C31.987 30.9664 32.9554 31.2608 33.9204 31.5657C34.3169 31.6908 34.5647 32.0742 34.4996 32.4515C34.4312 32.852 34.0938 33.1404 33.5518 33.1431H33.5525Z"
                fill="#02322C"
              />
              

              <path
                d="M26.4093 30.0321C26.2333 29.9544 26.042 29.8991 25.8839 29.795C25.0385 29.2391 24.1989 28.6734 23.3608 28.107C22.9682 27.8416 22.862 27.3951 23.0911 27.0157C23.3256 26.6278 23.8144 26.4822 24.2129 26.7391C25.1368 27.3351 26.0473 27.9529 26.9513 28.5786C27.2322 28.7729 27.3119 29.1851 27.187 29.5006C27.0701 29.795 26.7739 29.9847 26.4093 30.0321Z"
                fill="#02322C"
              />
              

              <path
                d="M25.5889 3.95118C28.5465 3.94723 30.9375 6.32083 30.9368 9.26018C30.9368 12.1864 28.5524 14.5672 25.6215 14.5698C22.6414 14.5725 20.2457 12.2213 20.2444 9.29179C20.2431 6.31491 22.6048 3.95513 25.5889 3.95118ZM25.5756 12.9438C27.6439 12.947 29.3103 11.3104 29.3229 9.26413C29.3348 7.25407 27.6173 5.54697 25.5876 5.55092C23.5572 5.55554 21.8742 7.21917 21.859 9.23581C21.8437 11.2762 23.5121 12.9398 25.5756 12.9431V12.9438Z"
                fill="#02322C"
              />
              

            </g>
            

            <defs>
              

              <clippath
                id="clip0_807_249"
              >
                

                <rect
                  fill="white"
                  height="44"
                  width="51"
                />
                

              </clippath>
              

            </defs>
            

          </svg>
          

        </span>
      </div>
      <div
        class="_location__divider_3ac954"
        style="background-color: rgba(0, 0, 0, 0.87);"
      />
      <div
        class="_location__section_3ac954"
      >
        <p
          class="MuiTypography-root MuiTypography-body2 _location__text_3ac954 css-1o4rdrb-MuiTypography-root"
        >
          RECURSOS
        </p>
        <span
          aria-hidden="true"
          class="_location__icon_3ac954"
        >
          <svg
            fill="none"
            height="49"
            viewBox="0 0 49 49"
            width="49"
            xmlns="http://www.w3.org/2000/svg"
          >
            

            <path
              d="M22.9778 0C23.9982 0 25.0187 0 26.0391 0C26.1622 0.0268198 26.2836 0.0694159 26.4066 0.0788817C30.0862 0.354968 33.5434 1.38358 36.7136 3.27044C43.2274 7.1451 47.2209 12.8135 48.6246 20.2789C48.7918 21.1687 48.877 22.0727 49 22.9704V26.0325C48.9716 26.1714 48.9369 26.3102 48.9164 26.4506C48.7745 27.4603 48.6972 28.4826 48.489 29.4765C45.7305 42.6387 32.9851 51.1075 19.8234 48.5422C7.4124 46.1221 -1.18649 34.5581 0.133627 22.0411C0.691956 16.7419 2.73128 12.0784 6.30521 8.12481C10.0858 3.94251 14.7653 1.33626 20.3344 0.369166C21.2113 0.217713 22.0961 0.121478 22.9778 0ZM27.3482 19.6763C27.3482 17.9945 27.2693 16.3664 27.3719 14.7509C27.4618 13.3389 28.2551 12.6305 29.6683 12.5422C30.8591 12.4665 32.0577 12.4917 33.2517 12.4744C33.4378 12.4712 33.6223 12.4744 33.8305 12.4744V6.52194C31.9079 6.52194 30.0105 6.43675 28.1226 6.53929C23.9241 6.76647 21.123 9.48158 20.7807 13.6718C20.6341 15.4624 20.7098 17.2719 20.6845 19.072C20.6814 19.2582 20.6845 19.4443 20.6845 19.6889H15.2022C15.1817 19.8056 15.1612 19.8656 15.1612 19.9271C15.1596 21.8565 15.1659 23.786 15.1517 25.7154C15.1486 26.0893 15.3568 26.0736 15.6107 26.0736C17.1248 26.0704 18.6389 26.0736 20.1546 26.0736C20.3234 26.0736 20.4905 26.0736 20.6956 26.0736V42.4668H27.2883V26.0814H33.5923V19.6778H27.3514L27.3482 19.6763Z"
              fill="#02322C"
            />
            

          </svg>
          

        </span>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
